recipes/conda-mirror/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "conda-mirror" %}
+{% set version = "0.6.5" %}
+{% set sha256 = "b609751148ffb5a1d36011d558673cb5eedf71327def05ea667c536004c1b0ca" %}
+
+{% set url_name = name.replace("-", "_") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ url_name[0] }}/{{ url_name }}/{{ url_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - conda-mirror = conda_mirror.conda_mirror:cli
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - requests
+    - pyyaml
+
+test:
+  imports:
+    - conda_mirror
+
+  commands:
+    - conda-mirror --help
+
+about:
+  home: https://github.com/ericdill/conda-mirror
```

Comment:
That is definitely the wrong url. Thanks for the fix here. Merging the PR that fixes this